app/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
 
     defaultConfig {
         applicationId "com.simplemobiletools.smsmessenger"
-        minSdkVersion 22
+        minSdkVersion 24
```

Comment:
why have you increased this?

Reply:
![image](https://user-images.githubusercontent.com/14886022/200187303-86847a1d-86d2-4890-adfe-4c4e45af5f14.png)
^in `Context.kt` this is what happens when the `minSdkVersion` is 22.

Reply:
thats not how you should solve it...

Reply:
Could you tell me how I would solve it or point me to relevant documentation? As I said, I'm new to the android ecosystem.

Reply:
well, if you cannot fetch the proper value in some other way and there is no other way to solve it, you should just disable the whole feature for the given SDK versions